src/platforms/Dashboards/DashboardNoordoostpolderProviderPlatform.js

```diff
@@ -0,0 +1,33 @@
+const platform = require('./_DashboardBasePlatform').clone();
+const destPath = '../dist/forus-platform.provider.noordoostpolder';
+
+// change platform name
+platform.setName('dashboard_noordoostpolder_provider');
+
+// tweaking output and cleaned paths config
+platform.setDest(`${destPath}`);
```

Comment:
tested this on dev.
Could you please check this file and update them with the 
platform.setDestRootPath(destPath); functionality

Reply:
same for dashboards